website/docs/tutorials/08-cairo-1-support/03-contracts.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_label: Contracts
+---
+
+# Prepare contracts and libraries
+
+If you want to [build](./04-compiling.md) or [test](./02-testing.md) Cairo 1 contracts, you have to properly prepare them. That also applies to the libraries that you want to use in other places. There are three files needed for that:
+- `main.cairo` (example name) cairo file with the source code
+- `lib.cairo` file that will have a module declaration
+- `cairo_project.toml` file with the `[crate_roots]` section containing the information about the location of the library
```

Comment:
This info is duplicated in [this pr](https://github.com/software-mansion/protostar/pull/1785/files#diff-e4efa3f5979fdc9c8972e40725e07730f00bbf1ba7c6f050ff7c8ad2177ff38fR47)

Reply:
This is because this pr is a month old and the one you linked is a week old. Ok, I will delete this file.